projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -9,7 +9,12 @@
 use Automattic\Jetpack\Sync\Sender;
 use Automattic\Jetpack\Sync\Settings;
 
-// POST /sites/%s/sync
+// phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound
```

Comment:
I'd space this out from the adjacent class comment
```suggestion
// phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound

```
And maybe also apply it to the other files where you have this problem.